catroid/src/main/java/org/catrobat/catroid/content/Script.java

```diff
@@ -162,14 +154,6 @@ public void removeBrick(Brick brick) {
 		return brickList;
 	}
 
-	public void setPaused(boolean paused) {
-		this.paused = paused;
-	}
-
-	public boolean isPaused() {
-		return paused;
-	}
```

Comment:
`isPaused` is only used in tests.
The paused member itself is private and nowhere else read.